### PR TITLE
Adding a dev stream to  the generated YAML.

### LINKED ISSUE #103: incorrect channel in v1.1.0/generated/installer/ibm-spectrum-scale-csi-operator.yaml
**Describe the bug**
it has ibm-spectrum-scale  it should be ibm-spectrum-scale-dev

```
grep image: ibm-spectrum-scale-csi-operator.yaml
        image: quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v1.1.0
        image: quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v1.1.0

[root@c943f6n01-pvt v1.1.0-feb21]# sed -i  "s:ibm-spectrum-scale/:ibm-spectrum-scale-dev/:g" generated/installer/ibm-spectrum-scale-csi-operator.yaml
sed: can't read generated/installer/ibm-spectrum-scale-csi-operator.yaml: No such file or directory
[root@c943f6n01-pvt v1.1.0-feb21]# sed -i  "s:ibm-spectrum-scale/:ibm-spectrum-scale-dev/:g" ibm-spectrum-scale-csi-operator.yaml
[root@c943f6n01-pvt v1.1.0-feb21]# grep image: ibm-spectrum-scale-csi-operator.yaml
        image: quay.io/ibm-spectrum-scale-dev/ibm-spectrum-scale-csi-operator:v1.1.0
        image: quay.io/ibm-spectrum-scale-dev/ibm-spectrum-scale-csi-operator:v1.1.0

```
**To Reproduce**
1. ...

**Expected behavior**

**Environment**
Please run the following an paste your output here:
``` bash
# Developement
operator-sdk version 
go version

# Deployment
kubectl version
rpm -qa | grep gpfs
```
**Screenshots**
```
[root@c943f6n01-pvt v1.1.0-feb21]# oc describe  pod/ibm-spectrum-scale-csi-operator-5c64899cc7-kbvdw -n ibm-spectrum-scale-csi-driver
Name:         ibm-spectrum-scale-csi-operator-5c64899cc7-kbvdw
Namespace:    ibm-spectrum-scale-csi-driver
Priority:     0
Node:         worker10.ocp4-c1.pok.stglabs.ibm.com/172.29.0.195
Start Time:   Fri, 21 Feb 2020 13:22:13 -0500
Labels:       app.kubernetes.io/instance=ibm-spectrum-scale-csi-operator
              app.kubernetes.io/managed-by=ibm-spectrum-scale-csi-operator
              app.kubernetes.io/name=ibm-spectrum-scale-csi-operator
              name=ibm-spectrum-scale-csi-operator
              pod-template-hash=5c64899cc7
              product=ibm-spectrum-scale-csi
              release=ibm-spectrum-scale-csi-operator
Annotations:  k8s.v1.cni.cncf.io/networks-status:
                [{
                    "name": "openshift-sdn",
                    "interface": "eth0",
                    "ips": [
                        "10.20.12.13"
                    ],
                    "dns": {},
                    "default-route": [
                        "10.20.12.1"
                    ]
                }]
              openshift.io/scc: anyuid
              productID: ibm-spectrum-scale-csi-operator
              productName: IBM Spectrum Scale CSI Operator
              productVersion: 1.1.0
Status:       Pending
IP:           10.20.12.13
IPs:
  IP:           10.20.12.13
Controlled By:  ReplicaSet/ibm-spectrum-scale-csi-operator-5c64899cc7
Containers:
  ansible:
    Container ID:
    Image:         quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v1.1.0
    Image ID:
    Port:          <none>
    Host Port:     <none>
    Command:
      /usr/local/bin/ao-logs
      /tmp/ansible-operator/runner
      stdout
    State:          Waiting
      Reason:       ErrImagePull
    Ready:          False
    Restart Count:  0
    Liveness:       exec [./health_check.sh] delay=10s timeout=1s period=30s #success=1 #failure=3
    Readiness:      exec [./health_check.sh] delay=3s timeout=1s period=1s #success=1 #failure=3
    Environment:    <none>
    Mounts:
      /tmp/ansible-operator/runner from runner (ro)
      /var/run/secrets/kubernetes.io/serviceaccount from ibm-spectrum-scale-csi-operator-token-j66w7 (ro)
  operator:
    Container ID:
    Image:          quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v1.1.0
    Image ID:
    Port:           <none>
    Host Port:      <none>
    State:          Waiting
      Reason:       ImagePullBackOff
    Ready:          False
    Restart Count:  0
    Liveness:       exec [./health_check.sh] delay=10s timeout=1s period=30s #success=1 #failure=3
    Readiness:      exec [./health_check.sh] delay=3s timeout=1s period=1s #success=1 #failure=3
    Environment:
      WATCH_NAMESPACE:  ibm-spectrum-scale-csi-driver (v1:metadata.namespace)
      POD_NAME:         ibm-spectrum-scale-csi-operator-5c64899cc7-kbvdw (v1:metadata.name)
      OPERATOR_NAME:    ibm-spectrum-scale-csi-operator
    Mounts:
      /tmp/ansible-operator/runner from runner (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from ibm-spectrum-scale-csi-operator-token-j66w7 (ro)
Conditions:
  Type              Status
  Initialized       True
  Ready             False
  ContainersReady   False
  PodScheduled      True
Volumes:
  runner:
    Type:       EmptyDir (a temporary directory that shares a pod's lifetime)
    Medium:
    SizeLimit:  <unset>
  ibm-spectrum-scale-csi-operator-token-j66w7:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  ibm-spectrum-scale-csi-operator-token-j66w7
    Optional:    false
QoS Class:       BestEffort
Node-Selectors:  <none>
Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
                 node.kubernetes.io/unreachable:NoExecute for 300s
Events:
  Type     Reason     Age                From                                           Message
  ----     ------     ----               ----                                           -------
  Normal   Scheduled  <unknown>          default-scheduler                              Successfully assigned ibm-spectrum-scale-csi-driver/ibm-spectrum-scale-csi-operator-5c64899cc7-kbvdw to worker10.ocp4-c1.pok.stglabs.ibm.com
  Warning  Failed     35s (x3 over 60s)  kubelet, worker10.ocp4-c1.pok.stglabs.ibm.com  Error: ImagePullBackOff
  Normal   BackOff    35s (x5 over 60s)  kubelet, worker10.ocp4-c1.pok.stglabs.ibm.com  Back-off pulling image "quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v1.1.0"
  Warning  Failed     35s (x5 over 60s)  kubelet, worker10.ocp4-c1.pok.stglabs.ibm.com  Error: ImagePullBackOff
  Normal   BackOff    35s (x3 over 60s)  kubelet, worker10.ocp4-c1.pok.stglabs.ibm.com  Back-off pulling image "quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v1.1.0"
  Warning  Failed     20s (x3 over 60s)  kubelet, worker10.ocp4-c1.pok.stglabs.ibm.com  Failed to pull image "quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v1.1.0": rpc error: code = Unknown desc = Error reading manifest v1.1.0 in quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator: manifest unknown: manifest unknown
  Warning  Failed     20s (x3 over 60s)  kubelet, worker10.ocp4-c1.pok.stglabs.ibm.com  Error: ErrImagePull
  Normal   Pulling    20s (x3 over 60s)  kubelet, worker10.ocp4-c1.pok.stglabs.ibm.com  Pulling image "quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v1.1.0"
[root@c943f6n01-pvt v1.1.0-feb21]#
```

**Additional context**

### PR DESCRIPTION
**Pull request by mew2057**
Fixes #103

Creates `generated/installer/ibm-spectrum-scale-csi-operator-dev.yaml` which points to the dev version of the release.

### CODE REVIEW

**Comment by mew2057** [comment]
This doesn't affect any running operator code, it's just an alternate config file pointing to the dev stream.

**Comment by Jainbrt** [comment]
Hi John, what is ETA for this PR to be merged in the 1.1.0,  Test team is blocked for this hence test cycle is getting delayed too?

**Comment by Jainbrt** [comment]
And one more clarity, whether we will have this dev yaml file for every release going forward for testing only and for customer this will not be published ?

File: generated/installer/ibm-spectrum-scale-csi-operator.yaml [hunk 1]
```diff
@@ -65,7 +65,11 @@ spec:
         - mountPath: /tmp/ansible-operator/runner
           name: runner
           readOnly: true
-      - env:
+        env:
+        - name: CSI_DRIVER_IMAGE
+          value: quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-driver:v1.1.0
```
Comment (smitaraut) [inline generated/installer/ibm-spectrum-scale-csi-operator.yaml]: Does this become another place where the driver image is baked into operator in addition to "roles/csi-scale/defaults/main.yml". How does the end user set  CSI_DRIVER_IMAGE?

Comment (mew2057) [reply generated/installer/ibm-spectrum-scale-csi-operator.yaml]: No, this  is the solution to the bake in, now it's in a non compiled file  that we can  generate without an image redeploy.

**Comment by mew2057** [comment]
The Dev stream is  something  that any user consuming the dev stream can ingest. Right now  we're pointing at the `1.1.0` dev stream, when we bump versions  the version that this config  will consume will be bumped too.

**Comment by mew2057** [comment]
This depends on #110 to be merged, because I'm not merging this change with a  breaking travis build.

**Pull request merged**